### ISSUE TITLE
🐞[BUG] Instance desyncs from its files on name change

### ISSUE THREAD

**Issue by SalmonSnakeD** [modrinth/code#3021]
### The location data of the folder that's linked to an instance desyncs on name changes.

Attempting to manually change the folder name to the same as the instance name does nothing.
The minimum fix would be to allow us to manually edit the location of the instance files.
The optimal fix would be to dynamically change the associated folder name and location on name changes.
___

- [x] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate feature requests
- [x] I have checked that this feature request is not on our [roadmap](https://roadmap.modrinth.com)

> Related to this idea: https://github.com/modrinth/code/issues/2424

___

**Comment by BananaDogBananaDog** [comment]
i second this

**Comment by xCheezie** [comment]
same

**Comment by clicktypeclick** [comment]
yeah wtf is up with that. i changed the name of my modpack and now it's broken

**Comment by Prospector** [comment]
folder name cannot be changed
